File: 1-Basics/1.1-Patterns/Patterns_python/8-InvertedStarPyramid.py
n = int(input("Enter Row Size N : "))

for i in range(n,0,-1):

    # left space 
    for j in range(n-i):
        print(" ", end="")
    
    # star 
    for l in range((i*2)-1):
        print("*", end="")

    print("\n")

File: 1-Basics/1.1-Patterns/Patterns_cpp/8-InvertedStarPyramid.cpp
#include<bits/stdc++.h>
using namespace std;


void pattern(int n){
    for(int i=n; i>0; i--){

        // left space 
        for(int l=1; l<=n-i; l++){
            cout<<" ";
        }
        
        // star
        for(int j=1; j<=(i*2)-1; j++){
            cout<<"*";
        } 

        cout<<endl;
    }
}

int main(){
    cout<<"Enter Row Size N : ";
    int n;
    cin>>n;

    pattern(n);

    return 0;
}